### ISSUE TITLE
ERROR: CheckTx: (1): cannot delegate to non-existant PubKey

### ISSUE THREAD

**Issue by zhiqiang-bianjie** [cosmos/gaia#105]
I select a validation node from validators and delegate my token to him (call gaia client tx delegate). But there was an error "ERROR: CheckTx: (1): can not delegate to non-existant PubKey." However, some validation nodes are OK, which is why?

**Comment by rigelrozanski** [comment]
Does the validator show up in the when using `gaia client query candidates`? if not then you will not be able to delegate to that address

**Comment by zhiqiang-bianjie** [comment]
@rigelrozanski no,I get the validators list from "http://xxxx:46657/validators"。I try it again,when uses "gaia client query candidates",it's ok.why?

**Comment by rigelrozanski** [comment]
Basically there is an implementation bug in the current iteration of gaia where initialization validators in genesis are not properly setup and therefor cannot receive delegation, only candidates which have been initialized with `gaia client tx declare-candidacy` can be delegated too... it will not always be this way, this will be resolved soon

**Comment by zhiqiang-bianjie** [comment]
@rigelrozanski Ok,i see,thanks
